paper/paper.md

```diff
@@ -1,158 +1,140 @@
-
-## Who to blame and what to do
-
-
-
-
-
-
--
-
-## Subsection level 2
-
-Please keep sections to a maximum of only two levels.
-
-## Tables and figures
-
-Tables can be added in the following way, though alternatives are possible:
-
-Table: Note that table caption is automatically numbered and should be
-given before the table itself.
-
-| Header 1 | Header 2 |
-| -------- | -------- |
-| item 1 | item 2 |
-| item 3 | item 4 |
-
-A figure is added with:
-
-![Caption for BioHackrXiv logo figure](./biohackrxiv.png)
-
-# Other main section on your manuscript level 1
-
-Lists can be added with:
-
-1. Item 1
-2. Item 2
-
-# Citation Typing Ontology annotation
-
-You can use [CiTO](http://purl.org/spar/cito/2018-02-12) annotations, as explained in [this BioHackathon Europe 2021 write up](https://raw.githubusercontent.com/biohackrxiv/bhxiv-metadata/main/doc/elixir_biohackathon2021/paper.md) and [this CiTO Pilot](https://www.biomedcentral.com/collections/cito).
-Using this template, you can cite an article and indicate _why_ you cite that article, for instance DisGeNET-RDF [@citesAsAuthority:Queralt2016].
-
-The syntax in Markdown is as follows: a single intention annotation looks like
-`[@usesMethodIn:Krewinkel2017]`; two or more intentions are separated
-with colons, like `[@extends:discusses:Nielsen2017Scholia]`. When you cite two
-different articles, you use this syntax: `[@citesAsDataSource:Ammar2022ETL; @citesAsDataSource:Arend2022BioHackEU22]`.
-
-Possible CiTO typing annotation include:
-
-* citesAsDataSource: when you point the reader to a source of data which may explain a claim
-* usesDataFrom: when you reuse somehow (and elaborate on) the data in the cited entity
-* usesMethodIn
-* citesAsAuthority
-* citesAsEvidence
-* citesAsPotentialSolution
-* citesAsRecommendedReading
-* citesAsRelated
-* citesAsSourceDocument
-* citesForInformation
-* confirms
-* documents
-* providesDataFor
-* obtainsSupportFrom
-* discusses
-* extends
-* agreesWith
-* disagreesWith
-* updates
-* citation: generic citation
-
-
-# Results
-
-
-# Discussion
-
-...
-
-## Acknowledgements
-
-...
-
-## References
+# **Introduction**
+
+During the SWAT4HCLS 2024 hackathon, a group of participants from various
+research initiatives and consortia in bioimaging and medical imaging, including
+German BioImaging e.V, the Helmholtz Metadata Collaboration (HMC) Hub Health,
+the Division of Medical Image Computing at the German Cancer Research Center
+(DKFZ) and the EU Horizon project CCE\_DART (Data Rich Clinical Trials), came
+together.
+
+With 2024 being the 10-year anniversary of FAIR data, our discussion quickly
+came to revolve around the state of FAIR access to bioimaging data, and ideas
+how to improve it. Within the research bioimaging community, the Open
+Microscopy Environment (OME) Consortium has been working to standardize access
```

Comment:
should we include links to OME, etc.?

Reply:
Yes, what exactly should I cite? The candidates I have in mind are the 2003 "vision" paper and the 2012 "original" OMERO paper: https://www.openmicroscopy.org/citing-ome/. I incline more towards the 2003 vision.

Reply:
Definitely ok to use the original, though I meant even more just a footnote to the webpage (i.e. for present usage). Either way.